packages/dashmate/src/createDIContainer.js

```diff
@@ -111,6 +112,7 @@ async function createDIContainer() {
     migrateConfigFile: asValue(migrateConfigFile),
     isHelper: asValue(process.env.DASHMATE_HELPER === '1'),
     getConnectionHost: asClass(getConnectionHostFactory).singleton(),
+    ensureFileMountExists: asClass(ensureFileMountExistsFactory).singleton(),
```

Comment:
asFactory

Reply:
There is no `asFactory`, changed to `asFunction` instead